addon/package.json

```diff
@@ -18,7 +18,8 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
-    "idea-town": "^0.3.0"
+    "idea-town": "^0.3.0",
+    "xtend": "^4.0.1"
```

Comment:
FWIW, might not need this module - [we the built-in `Object.assign`, which seems to do the same thing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign).